pkg/plugin/probes-handler/probes_handler.go

```diff
@@ -0,0 +1,37 @@
+package probeshandler
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+)
+
+// Probe defines the json elements for the probesHandler response body content
+type Probe struct {
+	Version string
+}
+
+// NewProbesHandler registers liveliness and readinesss probes for /version endpoint
+func NewProbesHandler() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Header().Set("Content-Type", "application/json")
+		_, err := w.Write(getJSONContent())
+		if err != nil {
+			log.Printf("Writing failed: %v", err)
```

Comment:
the logger that is used here is something different than the one that is used in the rest of the project. Please, use the same one. 
For more information see:
https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/log/log.go
https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/plugin/plugin_init.go#L105
https://github.com/arquillian/ike-prow-plugins/blob/master/pkg/plugin/plugin_init.go#L81

in addition, when there is an error in the probe, then the level should be higher - I would use `error` for it